### PR TITLE
Update guides for update_all 

### PR DESCRIPTION
**Pull request by dacook**
### Summary

The description for update_all seems dangerous, and quite out of date. I've made a suggested change to address each.

### Other Information
N/A

### CODE REVIEW

File: guides/source/active_record_basics.md [hunk 1]
```diff
@@ -288,18 +288,12 @@ user = User.find_by(name: 'David')
...
-hand, you'd like to update several records in bulk, you may find the
-`update_all` class method useful:
+This is most useful when updating several attributes at once. 
 
-```ruby
-User.update_all "max_login_attempts = 3, must_change_password = 'true'"
-```
-
-This is the same as if you wrote:
+If you'd like to update several records in bulk without callbacks or validations, you can update the database directly using `update_all`
```
Comment (p8) [inline guides/source/active_record_basics.md:293]: We try to wrap documentation to [80/100 characters](https://github.com/rails/rails/pull/37885#discussion_r400931085)
```suggestion
If you'd like to update several records in bulk without callbacks or
validations, you can update the database directly using `update_all`
```

**Review by p8 — CHANGES_REQUESTED** [review]
Thanks @dacook !
This seems an improvement to me. I have just one small issue.
Could you squash your commits after making that change?

File: guides/source/active_record_basics.md (hunk 1 above)
Comment (dacook) [reply guides/source/active_record_basics.md]: Thanks, done in new squashed commit.

**Comment by dacook** [comment]
Hi @p8 , I've applied your suggestion and squashed all into one commit.

**Comment by p8** [comment]
Thanks @dacook !

**Pull request merged**